### ISSUE TITLE
updateLoginTimestamp() must be of the type integer, null given

### ISSUE THREAD

**Issue by donslund** [einpraegsam/femanager#94]
Hi

I am trying to use FE Manager and I have used the tutorial to make som extra fields. But when I am trying to create a new user I get this error: updateLoginTimestamp() must be of the type integer, null given, called in /var/www/www.domain.dk/www/typo3/sysext/core/Classes/Authentication/AbstractUserAuthentication.php on line 90

Is it a FE Manager thing or a TYPO3 thing? I can't find any answers anywhere.

**Comment by jgreth** [comment]
Same here... -.-

**Comment by donslund** [comment]
@jgreth Did you solve your problem yet?

**Comment by jgreth** [comment]
@donslund sry, no.

**Comment by JeppeDonslund** [comment]
@jgreth  Any changes? I still have the same problem.

**Comment by jgreth** [comment]
@JeppeDonslund sorry, no. Moved this task in my "do it later" section...

**Comment by schumiel** [comment]
I still have the same problem.

**Comment by JeppeDonslund** [comment]
I solved it and now I am looking for what I did.

My problem now is that new users is not saved in the fe_user table. I get an email with their details, but nothing is saved.

 

 

Jeppe

 

Fra: schumiel <notifications@github.com> 
Sendt: 10. januar 2019 19:53
Til: in2code-de/femanager <femanager@noreply.github.com>
Cc: JeppeDonslund <jeppe@donslund.net>; Mention <mention@noreply.github.com>
Emne: Re: [in2code-de/femanager] updateLoginTimestamp() must be of the type integer, null given (#94)

 

I still have the same problem.
